src/components/media-query/__tests__/useMediaQuery.test.js

```diff
@@ -11,6 +11,15 @@ type Props = {
     children?: React.node,
 };
 
+jest.mock('../../../styles/_sizes.media.scss', () => {
+    return {
```

Comment:
I don't think these will be needed if we use the `variables.ts` approach.

Reply:
Went with `variables.ts` approach so removed these mocks.